splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/RowCountOperationIT.java

```diff
@@ -69,11 +69,15 @@
     private TestConnection conn;
     private String connectionString;
 
+    private static final String CONTROL = "jdbc:splice://localhost:1527/splicedb;create=true;user=splice;password=admin";
```

Comment:
To ensure the execution goes through CONTROL path, do we need to add explicit "useSpark=false"?

Reply:
Good point, I added it.